source/retryable-writes/tests/README.rst

```diff
@@ -456,6 +456,50 @@ and sharded clusters.
                 mode: "off",
         })
 
+#. Test that in a sharded cluster writes are retried on a different mongos if
+   one available
+
+   This test MUST be executed against a sharded cluster that has at least two
+   mongos instances.
+
+    1. Ensure that a test is run against a sharded cluster that has at least two
+       mongos-es. If there are more than two mongos-es in the cluster, pick two to
+       test against.
+
+    2. Create a client per mongos using the direct connection, and configure fail
+       points on each of the picked mongos-es, so that each mongos raises
+       a retryable error once.
```

Comment:
I suggest noting not to set a failpoint error that is transient, according to the [specifications](https://github.com/mongodb/specifications/blob/master/source/retryable-reads/retryable-reads.rst#why-are-read-operations-only-retried-once-by-default). See other comment for more details.